README.md

```diff
@@ -44,6 +44,82 @@ docker build -t pca-assessment .
 We welcome contributions!  Please see [here](CONTRIBUTING.md) for
 details.
 
+## Assessment JSON Field Dictionary ##
+
+The following items are included in the assessment JSON as produced by the `pca-wizard`.
+An example assessment JSON can be found [here](src/assessment/sample_assessment.json).
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-------:|:--------:|
+| id | RV number. | string | | yes |
+| timezone | Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones. | string | | yes |
```

Comment:
You should include an example here and for pretty much every other field (when possible), e.g.
`Timezone name based on [pytz](http://pytz.sourceforge.net/) timezones (e.g. "US/Eastern").`